src/file_auto_expiry/utils/interface.py

```diff
@@ -74,11 +74,13 @@ def collect_expired_file_information(folder_path, save_file, scrape_time, expiry
             "path": path, # storing pathname so we keep it when we transfer the dictionary to jsonl
             "creators": [creator for creator in creators if isinstance(creator[1], int) and creator[1] > 0 and creator[1] == creator[2]],
             "expired": is_expired,
-            "time_variables": {
+            "folder_stats": {
                 "atime_datetime": str(datetime.datetime.fromtimestamp(atime)),
                 "ctime_datetime": str(datetime.datetime.fromtimestamp(ctime)),
                 "mtime_datetime": str(datetime.datetime.fromtimestamp(mtime)),
-                "days_unused": days_unused
+                "days_unused": days_unused,
+                "size_bytes": size,
+                "size_mb": size / BYTES_PER_MB
```

Comment:
```suggestion
                "size_MiB": size / BYTES_PER_MB
```

I'd use MiB, but if you want to keep everything lower case, this is fine too. Actually we might not even need this because we already have bytes and can convert easily. Is this for human readability?

Reply:
Just for readability yeah, I'm fine with just keeping it just as bytes and converting outside